samples/java/src/main/java/com/modulr/hmac/Hmac.java

```diff
@@ -0,0 +1,14 @@
+package com.modulr.hmac;
+
+import com.modulr.api.ModulrApiAuth;
+
+import java.util.Map;
+
+public class Hmac {
+    public static void main(String... args) {
+        ModulrApiAuth modulrAuth = new ModulrApiAuth("KNOWN-TOKEN", "SECRET-TOKEN");
+        Map<String, String> headers = modulrAuth.generateApiAuthHeaders("NONCE");
+
+        headers.forEach((key, value) -> System.out.println(key + ": " + value));
```

Comment:
Will this work correctly without setting the date? Doesn't seem to default to current date.

Reply:
generateHmac() is setting the date  to new Date() every time it is called. Is this what you've meant?